packages/app/src/app/pages/common/ErrorBoundary/CodeSadbox/buildCrashReport.ts

```diff
@@ -4,7 +4,7 @@ import VERSION from '@codesandbox/common/lib/version';
 interface IbuildCrashReport {
   error?: Error;
   trace?: string;
-  errorCode: string | null;
+  errorCode?: string | null;
```

Comment:
Not needed as long as `errorCode` is given to the component again
```suggestion
  errorCode: string | null;
```